test/integration/create-next-app/index.test.js

```diff
@@ -81,4 +81,77 @@ describe('create next app', () => {
       fs.existsSync(path.join(cwd, projectName, '.gitignore'))
     ).toBeTruthy()
   })
+
+  it('should allow example with GitHub URL', async () => {
+    const projectName = 'github-app'
+    const res = await run(
+      projectName,
+      '--example',
+      'https://github.com/zeit/next-learn-demo/tree/master/1-navigate-between-pages'
```

Comment:
This test is working now but will fail if we update https://github.com/zeit/next-learn-demo to have different example names (e.g. if we rename `1-navigate-between-pages` or remove `about.js`). Should we create another repository just for testing purposes?

Reply:
We should use zeit/next.js imo.